clients/sender/controllers/main.html

```diff
@@ -1 +1,6 @@
-<h3>This is the main page</h3>
+<h3>Card Cast</h3>
+
+<form method="get" action="sendMessage()">
+  <textarea id="textBox" rows="10" cols="80">Your markdown here</textarea>
+  <button>Submit</button>
```

Comment:
This needs to be `<input type="submit" name="Submit" />` so that it has the ability to trigger the form and thus the `ng-submit` function.